lib/app/docker/distribution.go

```diff
@@ -120,7 +120,10 @@ func alive(path string, handler http.Handler) http.Handler {
 			return
 		}
 
-		handler.ServeHTTP(w, r)
+		ctx, cancel := defaultContext()
```

Comment:
This fixes an issue with distribution always logging at the default (`Debug`) level in HTTP handlers.